<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorBrightnessWidget</class>
 <widget class="QWidget" name="InspectorBrightnessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBrightness">
     <item>
      <spacer name="horizontalSpacerBrightness">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>9</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelBrightness">
       <property name="text">
        <string>&lt;a href=&quot;Brightness&quot; style=&quot;color: white;&quot;&gt;Brightness&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderBrightness">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBrightness">
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDuration">
     <item>
      <spacer name="horizontalSpacerDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDuration">
       <property name="text">
        <string>&lt;a href=&quot;Duration&quot; style=&quot;color: white;&quot;&gt;Duration&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTween">
       <property name="text">
        <string>&lt;a href=&quot;Tween&quot; style=&quot;color: white;&quot;&gt;Tween&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDefer">
     <item>
      <spacer name="horizontalSpacerDefer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>29</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDefer">
       <property name="text">
        <string>&lt;a href=&quot;Defer&quot; style=&quot;color: white;&quot;&gt;Defer&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDefer">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDefer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerBrightness">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>labelDuration</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>resetDuration(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTween</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>resetTween(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>durationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelBrightness</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>resetBrightness(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>sliderBrightnessChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>spinBoxBrightnessChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDefer</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>resetDefer(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDefer</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorBrightnessWidget</receiver>
   <slot>deferChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sliderBrightnessChanged(int)</slot>
  <slot>resetBrightness(QString)</slot>
  <slot>resetDuration(QString)</slot>
  <slot>resetTween(QString)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>durationChanged(int)</slot>
  <slot>spinBoxBrightnessChanged(int)</slot>
  <slot>resetDefer(QString)</slot>
  <slot>deferChanged(int)</slot>
 </slots>
</ui>
